crates/hir_def/src/diagnostics.rs

```diff
@@ -86,3 +86,28 @@ impl Diagnostic for UnresolvedImport {
         true
     }
 }
+
+// Diagnostic: unconfigured-code
+//
+// This diagnostic is shown for code with inactive `#[cfg]` attributes.
+#[derive(Debug)]
+pub struct UnconfiguredCode {
```

Comment:
Maybe call this "inactive"?

Reply:
or "inert"?

Reply:
To clarify, I think we need a shorter catchy internal name, and inert fills the bill. For public-facing message, we need something longer